src/apps/replication/client_lib/replication_common.h

```diff
@@ -43,16 +43,6 @@ using namespace ::dsn::service;
 
 namespace dsn { namespace replication {
 
-inline bool operator < (const global_partition_id& l, const global_partition_id& r)
```

Comment:
fixed in https://github.com/qinzuoyan/rDSN/commit/6f6bdfa1b939185f4e55a6252d34e7e092878f8c

but another problem is: higher user who include <replication.h> can not see these methods, or they may not depending on them, or they may write these methods by themselves. you can see <src/rep_tests/simple_kv/common.h>.